tests/data/dsl-semantic-validation-cases.yaml

```diff
@@ -741,16 +741,16 @@
         define parent: [document, document:*]
         define viewer: [user] or viewer from parent
   expected_errors:
-    - msg: 'type restriction `document:*` cannot contain both wildcard and relation'
+    - msg: '`parent` relation used inside from allows only direct relation.'
       line:
         start: 7
         end: 7
       column:
-        start: 1
-        end: 11
+        start: 42
+        end: 48
       metadata:
-        symbol: "document:*"
-        errorType: type-wildcard-relation
+        symbol: "parent"
+        errorType: tupleuset-not-direct
```

Comment:
```suggestion
        errorType: tupleuserset-not-direct
```

Reply:
I figured this was a typo on the enum, are we ok to replace throughout the whole project?

Reply:
Done in db74f853f558b62cd0181957231a50b8aa130c13